vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -559,6 +559,8 @@ func startDeploymentPods(f *framework.Framework, deployment *appsv1.Deployment)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	gomega.Expect(afterScale.Spec.Replicas).To(gomega.Equal(desiredPodCount), fmt.Sprintf("expected %d replicas after scaling", desiredPodCount))
 
+	deployment, err = c.AppsV1().Deployments(ns).Get(deployment.Name, metav1.GetOptions{})
```

Comment:
Please add a comment explaining why we do this. We had the bug for a long time and nobody noticed so I think it's not obvious

Reply:
Done.